pkg/util/wait/wait_test.go

```diff
@@ -12,13 +12,16 @@ import (
 	"testing"
 	"time"
 
+	"github.com/sirupsen/logrus"
+
```

Comment:
surprised this newline isn't causing a validate error

Reply:
https://github.com/openshift/openshift-azure/pull/839

Reply:
will remove newline.